Documentation/ExtensionArchitecture/Extbase/Reference/_FileUpload/_BlogEnhanced.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Domain\Model;
+
+use TYPO3\CMS\Core\Resource\Enum\DuplicationBehavior;
+use TYPO3\CMS\Extbase\Annotation\FileUpload;
+use TYPO3\CMS\Extbase\Domain\Model\FileReference;
+use TYPO3\CMS\Extbase\DomainObject\AbstractEntity;
+use TYPO3\CMS\Extbase\Persistence\ObjectStorage;
+
+class Blog extends AbstractEntity
+{
+    // A single file
+    #[FileUpload([
+        'validation' => [
+            'required' => true,
+            'maxFiles' => 1,
+            'fileSize' => ['minimum' => '0K', 'maximum' => '2M'],
+            'allowedMimeTypes' => ['image/jpeg'],
+            'imageDimensions' => ['maxWidth' => 4096, 'maxHeight' => 4096],
+        ],
+        'uploadFolder' => '1:/user_upload/extbase_single_file/',
+        'addRandomSuffix' => false,
```

Comment:
Let us always use `addRandomSuffix` => true,` in examples for security reasons.

Reply:
done